src/js/components/hocs.js

```diff
@@ -10,36 +9,25 @@ export const withFocus = (WrappedComponent) => {
   class FocusableComponent extends Component {
     constructor() {
       super();
-      this.handleActiveMouse = this.handleActiveMouse.bind(this);
       this.setFocus = this.setFocus.bind(this);
       this.state = {
         mouseActive: false,
         focus: false,
       };
     }
-    componentDidMount() {
-      window.addEventListener('mousedown', this.handleActiveMouse);
 
-      // we could be using onFocus in the wrapper node itself
-      // but react does not invoke it if you programically
```

Comment:
where did we move this to?

Reply:
I just validated the focus stopped working

Reply:
What is a scenario I can use to reproduce the issue?

Reply:
I figured it out. I've restored the behavior.